src/schema.jl

```diff
@@ -176,6 +176,8 @@ concrete_term(t::Term, d) = concrete_term(t, d, nothing)
 concrete_term(t, d, hint) = t
 
 concrete_term(t::Term, xs::AbstractVector{<:Number}, ::Nothing) = concrete_term(t, xs, ContinuousTerm)
+# and for missing values
+concrete_term(t::Term, xs::AbstractVector{Union{Missing,T}} where T<:Number, ::Nothing) = concrete_term(t, xs, ContinuousTerm)
```

Comment:
Couldn't those lines be
```
concrete_term(t::Term, xs::AbstractVector{<:Union{<:Number,<:Union{Missing,<:Number}}}) =
    concrete_term(t, xs, ContinuousTerm)
```
I don't think we need/should specialize.

Reply:
Wouldn't just having `AbstractVector{<:Union{Missing, <:Number}}` work?

```
julia> [1,2] isa AbstractVector{<:Union{Missing,<:Number}}
true

julia> [1,2, missing] isa AbstractVector{<:Union{Missing,<:Number}}
true
```